File: Backend/PF3311.TeleRehab.Api/Models/Patient.cs
namespace PF3311.Telerehab.API.Models;

public class Patient
{
    public Guid Id { get; set; }
    public string FullName { get; set; } = string.Empty;
    public int Age { get; set; }
    public string Sex { get; set; } = string.Empty;
    public string Condition { get; set; } = string.Empty;
    public string TechnologyLevel { get; set; } = string.Empty;
}

File: App1/telerehab_app/lib/models/patient.dart
class Patient {
  final String id;
  final String fullName;
  final int age;
  final String sex;
  final String condition;
  final String technologyLevel;

  Patient({
    required this.id,
    required this.fullName,
    required this.age,
    required this.sex,
    required this.condition,
    required this.technologyLevel,
  });

  factory Patient.fromJson(Map<String, dynamic> json) {
    return Patient(
      id: json['id'] ?? '',
      fullName: json['fullName'] ?? '',
      age: json['age'] ?? 0,
      sex: json['sex'] ?? '',
      condition: json['condition'] ?? '',
      technologyLevel: json['technologyLevel'] ?? '',
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'fullName': fullName,
      'age': age,
      'sex': sex,
      'condition': condition,
      'technologyLevel': technologyLevel,
    };
  }
}